src/plugins/plugin.legend.js

```diff
@@ -429,8 +448,7 @@ export class Legend extends Element {
 		const fillText = function(x, y, legendItem, textWidth) {
 			const halfFontSize = fontSize / 2;
 			const xLeft = rtlHelper.xPlus(x, boxWidth + halfFontSize);
-			const yMiddle = y + halfFontSize;
-
+			const yMiddle = y + (Math.max(fontSize, boxHeight) / 2);
```

Comment:
Having `> fontSize` in `getBoxHeight` would remove quite a few Math.max calls here.
Or could cache the `Math.max(fontSize, boxHeight)` to another variable.

Reply:
I don't think that's the case in general. It would happen when `usePointStyle` is true, but when it's false, the boxHeight can still be larger than the font size.